vllm/inputs/data.py

```diff
@@ -301,19 +301,19 @@ def multi_modal_inputs(self) -> Union[Dict, "MultiModalKwargs"]:
         if inputs["type"] == "multimodal":
             return inputs.get("mm_kwargs", {})
 
-        assert_never(inputs)
+        assert_never(inputs)  # type: ignore[arg-type]
 
     @cached_property
     def multi_modal_hashes(self) -> List[str]:
         inputs = self.inputs
 
         if inputs["type"] == "token":
             return inputs.get("multi_modal_hashes", [])
+        elif inputs["type"] == "multimodal":
```

Comment:
```suggestion

       if inputs["type"] == "multimodal":
```
Avoid unnecessary line changes